app/utils.py

```diff
@@ -0,0 +1,103 @@
+import math
+import datetime
+from typing import Union
+from app.customer import Customer
+from app.shop import Shop
+
+
+def remove_trailing_zeros(num: float) -> float:
```

Comment:
why do you need it?? you can use `round`

Reply:
Because depending on the result, I need different answers, and if I use round, the results are displayed with zero after the decimal point, and in most tests, such values are not allowed.